pkg/apis/apim/v1alpha1/types.go

```diff
@@ -158,6 +158,7 @@ type APIManagerSpec struct {
 	Replicas       *int32             `json:"replicas"`
 	Profiles 	[]Profile  				`json:"profiles"`
 	Service Service `json:"service"`
+	UseVolume bool `json:"useVolume"`
```

Comment:
There is no need to introduce a new flag. When processing the custom pattern, you just need to ignore those volumes. Shall we do that change?

Reply:
I'll look into this.

Reply:
Fixed